docs/03-tutorials/00-api-exposure/apix-04-expose-and-secure-workload-jwt-istio-ext-auth.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Expose and secure a workload with Istio external auth
+---
+
+This tutorial shows how to expose workload using Istio authorization policy and secure it using Istio RequestAuthentication with JWT token
+
+## Prerequisites
+
+To follow this tutorial, use Kyma 2.0 or higher.
+
+This tutorial is based on a sample HttpBin service deployment and a sample Function. To deploy or create one of those, follow the [Create a workload](./apix-02-create-workload.md) tutorial.
+
+## Expose your workload using VirtualService
+
+1. Export environment variable with your namespace:
+
+   ```shell
+   export NAMESPACE={NAMESPACE} # e.g. default
+   ```
+
+1. Run:
+
+   ```shell
+   cat <<EOF | kubectl apply -f -
+   apiVersion: networking.istio.io/v1alpha3
+   kind: Gateway
+   metadata:
+     name: httpbin-gateway
+     namespace: istio-system
+   spec:
+     selector:
+       istio: ingressgateway
+     servers:
+     - port:
+         number: 80
+         name: http
+         protocol: HTTP
+       hosts:
+       - "*"
+   ---
+   apiVersion: networking.istio.io/v1alpha3
+   kind: VirtualService
+   metadata:
+     name: httpbin
+     namespace: $NAMESPACE
+   spec:
+     hosts:
+     - "*"
+     gateways:
+     - istio-system/httpbin-gateway
+     http:
+     - match:
+       - uri:
+           prefix: /
+       route:
+       - destination:
+           port:
+             number: 8000
+           host: httpbin.default.svc.cluster.local
+   EOF
```

Comment:
Is manual creation of virtualservice really needed?

Reply:
@dariusztutaj yes in this scenario we need to manually expose svc with vs